app/src/androidTest/java/com/veniosg/dir/test/acceptance/FileManagerActivityTest.java

```diff
@@ -38,6 +40,17 @@
     private final File testChildDirectory = new File(testDirectory, "testChildDir");
     private final File testCopyDestination = new File(testDirectory, "testCopyDestination");
     private final File testChildFile = new File(testDirectory, "testChildFile");
+    private final File testDirectoryWithZeroItems = new File(sdCardDir, "testDirWithZeroItems");
+    private final File testDirectoryWithOneItem = new File(sdCardDir, "testDirWithOneItem");
+    private final File testDirectoryWithMultipleItems = new File(sdCardDir, "testDirWithMultipleItems");
+
+    private final File zeroItemsContainerDir = new File(testDirectoryWithZeroItems, "zeroItemsContainerDir");
+    private final File oneItemContainerDir = new File(testDirectoryWithOneItem, "oneItemContainerDir");
+    private final File multipleItemsContainerDir = new File(testDirectoryWithMultipleItems, "multipleItemsContainerDir");
+
+    private File oneFile = null;
```

Comment:
Make final and initialize the same way as the rest.

Reply:
Maybe also name these three something more consistent like `countTestFile1/2/3`

Reply:
Done!